Sources/SwiftDocC/Utility/FoundationExtensions/String+Path.swift

```diff
@@ -18,13 +18,13 @@ extension StringProtocol {
     }
     
     /// A copy of the string appended with a slash ("/") if the string doesn't already end with a trailing slash.
-    var appendingTrailingSlash: String {
+    package var appendingTrailingSlash: String {
```

Comment:
So package scope includes the test code? At least the way Swift DocC is setup?

Reply:
Yes, package access is available to all targets in the package.